joomla_extensions_development.xml

```diff
@@ -8711,10 +8711,10 @@ class Example extends CMSPlugin implements SubscriberInterface
               allowed to add (create) a new item? By default, it only checks
               if the currently logged in user has the
               <literal>core.create</literal> privilege for your component or
-              they have the <literal>core.create</literal> privilege in any
-              categories which belong to your component (assuming you are
+              has the <literal>core.create</literal> privilege in any
+              categories that belong to your component (assuming you are
```

Comment:
Also wrong edit. I am writing —or at least try to write— in British English which uses "which" instead of "that". The latter is American English.